dev_tools/notebooks/notebook_test.py

```diff
@@ -87,7 +87,7 @@ def env_with_temporary_pip_target():
 
 
 @pytest.mark.slow
-@pytest.mark.skipif(sys.platform != "linux", reason="Linux-only test")
+@pytest.mark.skipif(sys.platform not in ["linux", "darwin"], reason="Linux/Mac-only test")
```

Comment:
```suggestion
@only_on_posix
```

please reuse the existing decorator `from dev_tools.test_utils import only_on_posix`
Otherwise LGTM, thank you!

Reply:
Done.